.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+        exclude_types: ["csv"]
+    -   id: end-of-file-fixer
+        exclude_types: ["json"]
+    -   id: check-yaml
+    -   id: check-added-large-files
+
+-   repo: https://github.com/pycqa/flake8
```

Comment:
This is currently the same config as vss-tools. One could argue that we possibly could clean up as some parts like flake8 is not relevant as we do not have code here. Opinions are welcome